packages/grafana-ui/src/components/TimePicker/_TimePicker.scss

```diff
@@ -7,6 +7,17 @@
   }
 }
 
+.active-time-picker {
+  border-color: $orange;
+  color: $orange;
+  &:focus {
+    color: $orange;
```

Comment:
do you need colors specified for every single state if the color is the same? Is this overwritten by some other selector?

Reply:
It needs to be specified for each state as otherwise it is overwritten by `.navbar-button:hover` and .`navbar-button:focus`.